README.md

```diff
@@ -15,18 +15,7 @@
 
 1. Include the library as a local library project.
 
-	```
-	allprojects {
-	   repositories {
-	      ...
-	      maven { url "https://jitpack.io" }
-	   }
-	}
-	```
-
-    ``` implementation 'com.github.yalantis:ucrop:2.2.9' ``` - lightweight general solution
-
-    ``` implementation 'com.github.yalantis:ucrop:2.2.9-native' ``` - get power of the native code to preserve image quality (+ about 1.5 MB to an apk size)
+    ``` implementation 'com.automattic:ucrop:2.2.10' ``` - lightweight general solution
 
 2. Add UCropActivity into your AndroidManifest.xml
```

Comment:
It's not a scope of this PR, but I think that the SDK should add this activity to their own manifest and rely on manifest merge to include it into client's manifest.